pkg/autoscaler/statserver/server_test.go

```diff
@@ -157,6 +155,10 @@ func assertReceivedOk(sm *autoscaler.StatMessage, statSink *websocket.Conn, stat
 	if !ok {
 		t.Fatalf("statistic not received")
 	}
+	if recv.Stat.Time == nil {
+		t.Fatalf("Stat time is nil")
+	}
+	sm.Stat.Time = recv.Stat.Time
```

Comment:
Should we rather exclude the `Time` field from the comparison, to make that more explicit?

Reply:
Ok, I ignored the `Time` field.